docs/source/release/0.61.0-notes.rst

```diff
@@ -432,6 +451,8 @@ Pull-Requests:
 * PR `#9774 <https://github.com/numba/numba/pull/9774>`_: Test on gpuCI with supported NumPy and Python versions (`gmarkall <https://github.com/gmarkall>`_)
 * PR `#9780 <https://github.com/numba/numba/pull/9780>`_: Default to turn off `sys.monitoring` (`sklam <https://github.com/sklam>`_)
 * PR `#9799 <https://github.com/numba/numba/pull/9799>`_: Fix regression in exception handling against 0.60. (`stuartarchibald <https://github.com/stuartarchibald>`_)
+* PR `#9821 <https://github.com/numba/numba/pull/9821>`_: Add debug print to debug test_monitoring_multiple_threads failing on win-02 (`sklam <https://github.com/sklam>`_)
+* PR `#9837 <https://github.com/numba/numba/pull/9837>`_: Replace "hard-errors" with NumbaErrors. (`stuartarchibald <https://github.com/stuartarchibald>`_)
```

Comment:
To recognize contribution related to the PR; otherwise the author name will not be associated with a PR.

```suggestion
* PR `#9837 <https://github.com/numba/numba/pull/9837>`_: Replace "hard-errors" with NumbaErrors. (`stuartarchibald <https://github.com/stuartarchibald>`_ `lericson <https://github.com/lericson>`_ for `#9816 <https://github.com/numba/numba/pull/9816>`_)
```